roles/kafka_controller/tasks/main.yml

```diff
@@ -278,6 +278,33 @@
   tags:
     - filesystem
 
+- name: Create log4j Directory
+  file:
+    path: "{{kafka_controller.log4j_file | dirname}}"
+    owner: "{{kafka_controller_user}}"
+    group: "{{kafka_controller_group}}"
+    state: directory
+    mode: '750'
+  tags:
+    - filesystem
+    - log
+  when: kafka_controller.log4j_file != kafka_broker.log4j_file
```

Comment:
May be we should compare to kafka package default path, thus `"{{ (base_path, 'etc/kafka/log4j.properties') | path_join }}"` ?